internal/apps/apps.go

```diff
@@ -56,3 +70,37 @@ func Validate(kind, app string, validExt []string) error {
 	}
 	return fmt.Errorf(msg.FileNotFound, app)
 }
+
+// Download downloads a file from remoteUrl to a temp directory and returns the path to the downloaded file
+func Download(remoteUrl string) (string, error) {
+	resp, err := http.Get(remoteUrl)
+
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		return "", fmt.Errorf("Unable to download app from %s: %s", remoteUrl, resp.Status)
```

Comment:
One day I'll plug the new improved linter back in, I promise 🙏 

```suggestion
		return "", fmt.Errorf("unable to download app from %s: %s", remoteUrl, resp.Status)
```